hypertrace-trace-enricher/enriched-span-constants/src/main/proto/org/hypertrace/traceenricher/enrichedspan/constants/v1/enriched_span_attribute.proto

```diff
@@ -19,6 +19,7 @@ enum Http {
     HTTP_REQUEST_QUERY_PARAM = 2 [(string_value) = "http.request.query.param"];
     HTTP_HOST = 3 [(string_value) = "HTTP_HOST"]; // We should probably stick to a standard of keeping all enriched attributes in capital letters.
     HTTP_REQUEST_PATH_PARAM = 4 [(string_value) = "http.request.path.param"];
+    HTTP_URL = 5 [(string_value) = "http.url"];
```

Comment:
`http.url` isn't an enriched attribute. It's a raw span attribute. There is no need to add it here